oShare/MultipeerConnectivityManager.swift

```diff
@@ -13,8 +13,8 @@ class MultipeerConnectivityManager: NSObject {
 	private var connectivityAdvertiser: MCNearbyServiceAdvertiser
 	
 	init(peer: MCPeerID) {
-		self.peer = peer
-		self.session = MCSession(peer: peer)
+		self.localPeer = peer
+		self.session = MCSession(peer: peer, securityIdentity: nil, encryptionPreference: .optional)
```

Comment:
This change is due to certain versions of iOS 10 and or 11, reportedly having `.optional` on some and `.required` on others. This prevented connections from successfully initiating.

See: https://stackoverflow.com/a/40392480